app/styles/_customization.scss

```diff
@@ -0,0 +1,3 @@
+.ember-basic-dropdown-content {
+  z-index: 1002;
```

Comment:
Can you check the solution used in the save template modal? I think we can deal with the z-index there instead of introducing a global like this